includes/Admin/Notices/Manager.php

```diff
@@ -189,4 +190,23 @@ private function dismiss_notice( string $option_name ) {
         update_option( $option_name, 'yes' );
         wp_send_json_success();
     }
+
+    public function show_admin_commission_notice( $notices ) {
+        if (
+            version_compare( DOKAN_PLUGIN_VERSION, '3.14.0', '>=' ) &&
+            defined( 'DOKAN_PRO_PLUGIN_VERSION' ) &&
+            version_compare( DOKAN_PRO_PLUGIN_VERSION, '3.14.0', '<' )
+        ) {
+            $notices[] = [
+                'priority'          => 1,
+                'show_close_button' => false,
+                'type'              => 'alert',
+                'scope'             => 'global',
+                'title'             => __( 'Upgrade to Dokan Pro Latest Version', 'dokan-lite' ),
+                'description'       => __( 'Your Dokan Lite version is updated to the latest version but you are still using the Dokan pro old version. Upgrade to Dokan Pro Latest version to enable the new features.', 'dokan-lite' ),
```

Comment:
Pls consult about the `title` and `description` with the @getdokan/product team.